docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Dockerfile
+FROM rust:1-stretch as builder
+# Choose a workdir
+WORKDIR /usr/src/app
+# Create blank project
+RUN USER=root cargo init
+# Copy Cargo.toml to get dependencies
+COPY Cargo.toml .
+# This is a dummy build to get the dependencies cached
+RUN cargo build --release
+# Copy sources
+COPY src src
+COPY contracts contracts
+RUN touch src/main.rs && cargo build --release
+
+FROM debian:stretch-slim
+RUN apt-get update && apt-get --yes install openssl
+COPY --from=builder /usr/src/app/target/release/etherbalance /bin/
+CMD etherbalance
```

Comment:
We should probably wrap this inside [tini](https://github.com/krallin/tini). We had an issue with our dex-services binary not handling signals well.

cf. https://github.com/gnosis/dex-services/blob/16ef8cf025c41b80318544df9a45890cb764deff/docker/rust/Dockerfile#L41-L44